adoption-metrics/define_adoption_metrics.md

```diff
@@ -0,0 +1,50 @@
+# What adoption metrics Packit team wants to measure
+
+## Static meaurement (updated periodically)
+
+### Newcomers adoption
+
+Newcomers are users who have an approved namespace.
+
+- Total amount of newcomers in a given period (monthly/quarterly/half yearly/yearly).
+
+### Active users
+
+Active users are those from whom we have recorded at least one activity in the past period
+and they do not fit into the category of newcomers.
+
+### Users retention
+
+- Total amount of active users.
+- Percentage of previous newcomers who became active Packit users.
+  ((amount of newcomers from previous period becoming an active users
+  / amount of newcomers from previous period) \* 100)
+- What percentage of all users are active users? (How many inactive users do we have?)
```

Comment:
this one will help with Franta's point from above right? when asking about installations that are not utilized

Reply:
Yes, we just need to clearly define who is an active user, newcomer, ...